packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -96,30 +119,35 @@ export const getStaticProps = async () => {
   <Column>
```

Comment:
(this comment is for BannerExample above)

todo: BannerExample should include borderless Status Badge, default Status Badge, and Status Menu